app/Models/Subscriber.php

```diff
@@ -67,7 +74,7 @@ public static function boot()
      *
      * @return \Illuminate\Database\Eloquent\Relations\HasMany
      */
-    public function subcriptions()
+    public function subscriptions()
```

Comment:
Can we have the typos and fixes merged seperately please so we can ship them with 2.1.1, and have the feature part going into 2.2.0.